src/Symfony/Component/Validator/ConstraintViolationList.php

```diff
@@ -158,4 +158,22 @@ public function offsetUnset($offset)
     {
         $this->remove($offset);
     }
+
+    /**
+     * Creates iterator for errors with specific codes.
+     *
+     * @param string|string[] $codes The codes to find
+     * @return static New instance which contains only specific errors.
+     */
+    public function findByCodes($codes)
+    {
+        $violations = array();
+        foreach ($this as $violation) {
+            if (in_array($violation->getCode(), $codes, true)) {
```

Comment:
@ro0NL I've added support passing code or array of codes. No need wrap single code with array